product_abc_classification/models/abc_classification_product_level.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class AbcClassificationProductLevel(models.Model):
+    _name = "abc.classification.product.level"
+    _inherit = "mail.thread"
+    _description = "Abc Classification Product Level"
+    _rec_name = "level_id"
+
+    display_name = fields.Char(compute="_compute_display_name")
```

Comment:
```suggestion

```
No need to add this field since it will be added by default (with `search` attribute)